train-1/HW2.cpp

```diff
@@ -0,0 +1,41 @@
+#include <opencv2/opencv.hpp>
+using namespace cv;
+
+#include <iostream>
+using namespace std;
+
+
+int main(int argc,char** argv) {
+    VideoCapture cap;    //创建存储视频文件或者设备的对象
+    cap.open(argv[1]);    //打开视频文件或者视频设备
+    if (!cap.isOpened())
+    {
+        cout << "could not open the VideoCapture !" << endl;
+        system("pause");
+        return -1;
+    }
+
+    const char* windowsName1 = "Original";
+    const char* windowsName2 = "Binary";
+    int k = -1;
+    while (true)
+    {
+        Mat frame,result;//读入帧和灰度图像
```

Comment:
U'd better make this line out of the `while-loop`.(I mean it's not good to initialize objects frequently, as `cv::Mat` is kind of "big object")